eng/pipelines/templates/jobs/smoke-tests.yml

```diff
@@ -65,48 +66,56 @@ jobs:
           TestTargetFramework: netcoreapp2.1
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
           ArmTemplateParameters: $(azureCloudArmParameters)
+          Location: "westus"
```

Comment:
Are we also hitting capacity issues with the default public cloud?

Reply:
No, this is just something we've been seeing occasionally in usgov for the past few months.